openedx/core/djangoapps/site_configuration/models.py

```diff
@@ -281,6 +285,7 @@ def compile_microsite_sass(self):
         return {
             'successful_sass_compile': successful_sass_compile,
             'sass_compile_message': sass_compile_message,
+            'scss_file_used': scss_file
```

Comment:
Great idea, thanks Esther for adding this variable.


Makes diff less annoying.

```suggestion
            'scss_file_used': scss_file,
```